juju/juju
# Fix upgrade step which remove orphaned saas proxies

An upgrade step removes any orphaned app proxies which exist in an offering model to represent the consuming side.
However, the filter to ignore the consuming side proxies was left off, so the saas applications were removed too.

## QA steps

deploy 2.8 models
make an offer and consume the offer
upgrade the controller and then the models to 2.9
check the cmr relations are in tact and the saas applications are green
